_posts/2020-05-14-status-update.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "14st May 2020: IAC Team Status Update"
+author: daianamezdrea
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - testing
+  - dsc
+  - pdk
+  - powershell
+  - windows
+  - modulebuilder
+  - voxpupuli
+---
+The IAC team has been working on a lot of interesting things over the past week. We want to share with our community some pretty interesting things. Here's a brief overview of what we've been up to:
+
+## COVID-19 who?
+We don't count anymore the days as a fully remote working team, but we count the days when we work hard and with all the energy we have for the best results!
+`#unstoppableteam #workHard #staysafe #thinkpositive`
+
+## Module Releases
+
+Modules released this week:
+- [puppetlabs/apt](https://github.com/puppetlabs/puppetlabs-apt) (7.4.2)
```

Comment:
Ciaran and David released two more modules postgresql and mysql.

Reply:
...and firewall too